aws-s3/latest/config/direct-access.md

```diff
@@ -0,0 +1,137 @@
+
+The `system.directAccessUrl.enabled` property is the main switch for this feature. If this is set to false, then **all** DAUs are disabled.
+
+The next configuration that controls specific DAUs is the one for the content store. The `connector.s3.directAccessUrl.enabled` property controls whether DAUs are enabled for that specific store.
+
+Whether or not a client can request a DAU by using a REST endpoint is controlled by the `restApi.directAccessUrl.enabled property`. If the REST endpoint is disabled, but the feature is enabled system-wide and on the content-store, then the DAUs will only be usable by Java clients (i.e. only service-level requests will be possible).
+
+## APIs
+
+### REST endpoints
+
+The following endpoints can be used to send requests to obtain DAUs in Content Services:
+
+* `POST /nodes/{nodeId}/request-direct-access-url`
+* `POST /nodes/{nodeId}/renditions/{renditionId}/request-direct-access-url`
+* `POST /nodes/{nodeId}/versions/{versionId}/request-direct-access-url`
+* `POST /deleted-nodes/{nodeId}/request-direct-access-url`
+* `POST /deleted-nodes/{nodeId}/renditions/{renditionId}/request-direct-access-url`
+
+Optionally, the POST body can specify an `attachment` flag. A value of `true` indicates that a download link is required; `false` indicates an embedded link is required. This defaults to `true` if it's not specified.
+
+The endpoints return the following response:
+
+```json
+{
+  "entry": {
+    "contentUrl": "<url>"
+    "attachment":"<boolean>"
+    "expiresAt": "<date-time>"
+    }
+}
+```
+
+The length of time for which a direct access URL is valid defaults to `300` seconds (i.e. 5 minutes) if not configured otherwise in `alfresco-global.properties`.
+
+<!-- method / description / request/ parameters / response / error codes -->
+
+**Method:** `POST`
+
+**Response:**
+
+Link to the resource wrapped in a JSON Object which also contains an attachment flag and the DAU expiration date.
+
+**Error codes:**
+
+If there's no DAU provider installed in Alfresco (such as the S3 Connector), or DAUs aren't enabled, then a `501` HTTP status code is returned.
+
+**Parameters:**
+
+* `attachment` is an optional flag which controls the download method (attachment URL vs. embedded URL). Defaults to `true` when not specified, which means the value of the [Content-Disposition](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition) response header will be attachment.
+* The `filename` part of the Content-Disposition header will be set in the service layer logic, and can't be controlled by the DAU client.
+
+### Discovery API
+
+The Discovery API provides status information about the DAUs feature (enabled/disabled) via a new field:
+
+* `RepositoryInfo > StatusInfo > isDirectAccessUrlEnabled`
```

Comment:
Maybe add example partial json snippet here (similar to earlier example when requesting DAU) ?